src/App.js

```diff
@@ -13,6 +13,7 @@ class App extends React.Component {
     done: false,
     operate: undefined,
     originalLastNum: "",
+    lastEquation: "",
```

Comment:
looks like you were trying to add `lastEquation` but ended up removing a setState. that happens a lot when refactoring! when it happens, I'll revert the other changes I originally tried to make so they can be included in the commit where I actually use them. Notice you don't use lastEquation yet. something to keep in mind that will make your commits easier to understand later.

Reply:
I removed 'done', 'operate', and 'originalLastNum', but I'm unclear on what you mean by I "ended up removing a setState"...

Reply:
Look at the commit with the message "remove state variables, update pressedEqual functions"

Reply:
Lol, whoops! The commit I was referring to was actually "remove one of onClick's unecessary setStates"

Reply:
Everything is great about that commit except it includes introducing `lastEquation` which feels like unexpected code.

Reply:
I introduced `lastEquation` and then updated it inside of `calculate`, `reset`, etc. because I wanted to start cutting out 'done', 'operate', and 'originalLastNum' but before doing so I wanted to introduce what would be replacing them (lastEquation) so that I could start seeing tests passing before removing them. I thought that was a best practice...to go slowly in changes and introduce the new variable before removing the one it will replace?

Reply:
going slowly and introducing the thing you'll switch to before deleting the existing. yes! that's great advice! but if you start one change and jump into another, it's a good idea to undo the first change. this makes each commit, a cohesive thought and easier to write a commit message for. you can either undo it when you realize you want to do something else first, or when you catch it in your diff before committing. I like to do the former.

Reply:
Ahhh okay, yes thank you for explaining! I will implement this best practice from now on.